CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog for `wai-saml2`
 
+## 0.4.1
+
+- Added `authnRequestDestination` field to `AuthnRequest`
```

Comment:
I think this probably warrants a major version change to comply with the PvP, but for now it is OK to just use "Unreleased changes" as the header here:

```suggestion
## Unreleased changes

- Added `authnRequestDestination` field to `AuthnRequest`
```

Reply:
I have made this change now and resolved the merge conflict.